internal/internal_activity.go

```diff
@@ -74,6 +74,7 @@ type (
 		DisableEagerExecution  bool
 		VersioningIntent       VersioningIntent
 		Summary                string
+		Priority               *commonpb.Priority
```

Comment:
What is the inheritance logic for `Priority` in a Workflow? If I call a workflow with a given priority and I don't set a priority when I start a child workflow or activity do they inherit the workflows?

Reply:
David's proto docs define it like this:
> // Priority is attached to workflows and activities. By default, activities
	// inherit Priority from the workflow that created them, but may override fields
	// when an activity is started or modified.

Yes, a child workflow would inherit the calling workflow's priority. Unless it overrides it.

For posterity, this is where that happens in server: https://github.com/temporalio/temporal/pull/7134/files#diff-bd46ccf32fa0605754c1426971b2e886c9a17d09ed7412474b51584ec7fee46fR904

Reply:
Ah OK It happens in the server, I wasn't sure where the logic was supposed to live.